### PR TITLE
Fix #422: Make MetricMultiProgressBar render Skeleton while data is loading to avoid layout shift.

### LINKED ISSUE #422: MetricMultiProgressBar - better loading state
Currently there's a layout shift when it renders.

On error I'm not sure what we should do.  It'll probably usually be rendered in conjunction with a value and so rendering nothing might be fine / expected.

### PR DESCRIPTION
**Pull request by mikelehen**
See https://act-now-packages--pr462-mikelehen-progressba-64no4ucj.web.app/storybook/index.html?path=/story/metrics-metricmultiprogressbar--loading-delay

### CODE REVIEW

File: packages/ui-components/src/components/MetricMultiProgressBar/MetricMultiProgressBar.tsx [hunk 1]
```diff
@@ -19,12 +20,14 @@ export interface MetricMultiProgressBarProps extends BaseMultiProgressBarProps {
 export const MetricMultiProgressBar = ({
   region,
   metrics,
+  width = 100,
+  height = 16,
```
Comment (mikelehen) [inline packages/ui-components/src/components/MetricMultiProgressBar/MetricMultiProgressBar.tsx]: These are the same defaults used by the ProgressBar component, but I'm now specifying them here so we can use them to size the Skeleton component.

File: packages/ui-components/src/components/MultiProgressBar/MultiProgressBar.tsx [hunk 2]
```diff
@@ -54,7 +54,13 @@ export const MultiProgressBar = <T,>({
   );
 
   return (
-    <svg width={width} height={height} aria-labelledby={titleId} role="meter">
+    <svg
+      width={width}
+      height={height}
+      aria-labelledby={titleId}
+      role="meter"
+      style={{ display: "block" }}
```
Comment (mikelehen) [inline packages/ui-components/src/components/MultiProgressBar/MultiProgressBar.tsx:67]: By default svg elements are inline-block which causes them to be rendered like text with some extra space below them based on your line-height, etc.  Displaying as "block" makes sure the exact size used by the svg will match the width/height passed (which we use in the skeleton component) and so we won't get layout shift.

File: packages/ui-components/src/components/MetricMultiProgressBar/MetricMultiProgressBar.tsx (hunk 1 above)
Comment (chasulin) [reply packages/ui-components/src/components/MetricMultiProgressBar/MetricMultiProgressBar.tsx]: nit - should we export these defaults from MultiProgressBar to ensure they stay in sync?

**Review by chasulin — APPROVED** [review]
looks good! [one small comment](https://github.com/covid-projections/act-now-packages/pull/462#discussion_r1036376382)

File: packages/ui-components/src/components/MetricMultiProgressBar/MetricMultiProgressBar.tsx (hunk 1 above)
Comment (mikelehen) [reply packages/ui-components/src/components/MetricMultiProgressBar/MetricMultiProgressBar.tsx]: Good flag, done!

**Pull request merged**